lib/active_model_serializers/adapter/base.rb

```diff
@@ -26,9 +26,7 @@ def serializable_hash(_options = nil)
       end
 
       def as_json(options = nil)
-        hash = serializable_hash(options)
-        include_meta(hash)
-        hash
+        serializable_hash(options)
       end
```

Comment:
It failed because the `alias` would be bound to the pre-inheritance placeholder method.